test/unit/function_export_mv_to_table_test.sql

```diff
@@ -2057,7 +2424,7 @@ select results_eq(
 
 
 -- Data in ggircs_swrs.measured_emission_factor === data in ggircs.measured_emission_factor
-select results_eq(
+select set_eq(
```

Comment:
:+1: This saves some headaches when we don't care about the order of the results. I suppose there could be some places where we do care about order, but not here. Much easier to add/change tests this way